app/models/events/new_discussion.rb

```diff
@@ -1,7 +1,8 @@
 class Events::NewDiscussion < Event
   def self.publish!(discussion)
     event = create!(kind: 'new_discussion',
-                    eventable: discussion)
+                    eventable: discussion,
+                    sequence_id: 0)
```

Comment:
How does it get a sequenceId of 40,000? I'm still confused and happy to chat in person about it.. but I don't understand .. The general idea is that only events with sequence id count towards the discussion thread... if this does not have a sequence id, then how does it sometimes get a very high one?